content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -110,6 +111,27 @@ either `Delete` or `Retain`. If no `reclaimPolicy` is specified when a
 Persistent Volumes that are created manually and managed via a storage class will have
 whatever reclaim policy they were assigned at creation.
 
+### Allow Volume Expansion
+
```

Comment:
Add a feature tag for beta

Reply:
@msau42  Thank you. I have added the feature state for the suggested comments.